mindmeld/text_preparation/tokenizers.py

```diff
@@ -242,8 +242,9 @@ def __init__(self, language, spacy_model_size="sm"):
             spacy_model_size (str, optional): Size of the Spacy model to use. ("sm", "md", or "lg")
         """
         self.spacy_model = SpacyModelFactory.get_spacy_language_model(
-            language, spacy_model_size
+            language, spacy_model_size, disable=["tagger", "parser", "ner"]
```

Comment:
Since the typical usage doesn't use any pipeline components, perhaps we should just be using `spacy.blank(language)`. This would mean apps wouldn't need to depend on the language model packages, but some languages would still need a tokenizer library, e.g. jieba for zh, pysudachi for ja.

Reply:
A thought: You can also disable the pipes for spacy models upon loading the model.
Example:
```
        self.spacy_model = SpacyModelFactory.get_spacy_language_model(
            language, spacy_model_size
        )
        self.spacy_model.disable_pipes('ner', 'tagger', 'parser')
```
This can help in not having to pass on the information along the hierarchy (pointing this because I don't think we are anyways storing the list of disable pipe names as an instance attribute in [mindmeld/text_preparation/spacy_model_factory.p](https://github.com/cisco/mindmeld/pull/398/files?diff=unified&w=1#diff-a248bdddac776431435cbd12e577828540f8eef0026f5e56eaf3b627b5b33b4a))

Reply:
Opting for the disable param option to retain functionality for selection a partial set of components

Reply:
Seems like we can also disable `lemmatizer` and `attribute_ruler`